.github/workflows/ci.yaml

```diff
@@ -325,11 +291,13 @@ jobs:
           kubectl cluster-info
           export KUBE_API=$(kubectl config view -o jsonpath='{.clusters[0].cluster.server}')
           kind get nodes
-          
+
       - name: Load Netchecks Images into Kind
         run: |
-          kind load docker-image ghcr.io/hardbyte/netchecks:edge
-          kind load docker-image ghcr.io/hardbyte/netchecks-operator:edge
+          docker pull ${{env.IMAGE_REGISTRY}}/${{env.IMAGE_REPOSITORY}}/${{env.PROBE_IMAGE_NAME}}:edge
+          docker pull ${{env.IMAGE_REGISTRY}}/${{env.IMAGE_REPOSITORY}}/${{env.OPERATOR_IMAGE_NAME}}:edge
+          kind load docker-image ${{env.IMAGE_REGISTRY}}/${{env.IMAGE_REPOSITORY}}/${{env.PROBE_IMAGE_NAME}}:edge
+          kind load docker-image ${{env.IMAGE_REGISTRY}}/${{env.IMAGE_REPOSITORY}}/${{env.OPERATOR_IMAGE_NAME}}:edge
```

Comment:
Hmm thoughts on using git sha instead of `edge`?

Reply:
Yeah it'd be better and would work with concurrent builds